### DISCUSSION TITLE
Forge

### DISCUSSION THREAD

**Discussion by Sensenmann01** [terrarium-earth/Ad-Astra#554] [General]
schad das es nicht für forg minecraft 1.20.4 ist, hätte den gern mal getestet aber so ne danke, nur wegen ein mod den loder wechseln lohnt sich nicht

**Comment by CodexAdrian** [comment]
We will not be supporting MinecraftForge going forward, please switch to NeoForge

**Comment by Sensenmann01** [comment]
warum überhaupt weg von forge, obwohl forge die meistem mods hat wen ich mein mod list so an schau (die um fast 294 mods) und nicht alle sind bei neo forge
________________________________
Von: Adrian O.V ***@***.***>
Gesendet: Dienstag, 14. Mai 2024 22:57
An: terrarium-earth/Ad-Astra ***@***.***>
Cc: Sensenmann01 ***@***.***>; Author ***@***.***>
Betreff: Re: [terrarium-earth/Ad-Astra] Forge (Discussion #554)


Closed #554<https://github.com/terrarium-earth/Ad-Astra/discussions/554> as resolved.

**Reply by CodexAdrian** [reply]
While there are some mods still on forge, most mod authors have committed to using Neoforge for their mod in 1.21. We personally no longer feel MinecraftForge is a viable platform on later versions, and would encourage others to use an alternative, like NeoForge

**Comment by Sensenmann01** [comment]
von den 295 mods die ich nutz sind grade mal 195 bei neoforge, und 42 mods sind reine forge mods, deswegen lohnt sich der wechsel überhaupt nicht
________________________________
Von: Adrian O.V ***@***.***>
Gesendet: Dienstag, 14. Mai 2024 23:08
An: terrarium-earth/Ad-Astra ***@***.***>
Cc: Sensenmann01 ***@***.***>; Author ***@***.***>
Betreff: Re: [terrarium-earth/Ad-Astra] Forge (Discussion #554)


While there are some mods still on forge, most mod authors have committed to using Neoforge for their mod in 1.21. We personally no longer feel MinecraftForge is a viable platform on later versions, and would encourage others to use an alternative, like NeoForge

**Reply by CodexAdrian** [reply]
Sorry to hear that, but this doesn't change much for us. We do not feel MinecraftForge will be a viable platform for modders in the future and will not continue developing mods for it. So you'll unfortunately have to switch to NeoForge or not include Ad Astra
